CHANGELOG-4.0.md

```diff
@@ -60,9 +60,10 @@
         - `Phalcon\Storage\Adapter\Stream`
     - `Phalcon\Storage\AdapterFactory`: Factory to create adapters
  [#13439](https://github.com/phalcon/cphalcon/issues/13439)
- - Changed `Phalcon\Mvc\Model\Query` to not call `numRows` when selecting data  [#13387](https://github.com/phalcon/cphalcon/issues/13387)
+- Changed `Phalcon\Mvc\Model\Query` to not call `numRows` when selecting data  [#13387](https://github.com/phalcon/cphalcon/issues/13387)
+- Changed `Phalcon\Validation\Validator\Url` to work with passed options. (`FILTER_FLAG_PATH_REQUIRED`, `FILTER_FLAG_QUERY_REQUIRED`) [#13548](https://github.com/phalcon/cphalcon/issues/13548)
 - `Phalcon\Mvc\Model\Query\Builder` will now omit offsets if they are set as 0.
- - `Phalcon\Mvc\Model\Transaction::__construct()` service parameter must be a string or omitted, `Phalcon\Logger\Formatter\Line::__construct()` parameters must be a string or omitted, `Phalcon\Logger\Formatter\Json::__construct()` parameters must be a string or omitted.
+- `Phalcon\Mvc\Model\Transaction::__construct()` service parameter must be a string or omitted, `Phalcon\Logger\Formatter\Line::__construct()` parameters must be a string or omitted, `Phalcon\Logger\Formatter\Json::__construct()` parameters must be a string or omitted.
```

Comment:
Can we have 3 log entries here instead of separating by comma?

Reply:
Resolved